File: DKPython/functions/dk_notice.py
import sys 
sys.path.append(".")
from DK import *

from dk_log import *

##################################################################################
# dk_notice(<message>)
#
#    Print a notice message to the console
#
#    @message - The message to print
#
def dk_notice(*args):
#    dk_call dk_debugFunc 1

    dk_log(NOTICE, args[0])





###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
def DKTEST():
#    dk_call dk_debugFunc 0

    dk_notice("test dk_notice message")

File: DKPhp/functions/dk_notice.php
<?php

################################################################################
# dk_notice(message)
#
#    Print a notice message to the console
#
#    @message - The message to print
#
function dk_notice() {
	#dk_debugFunc(1);
	dk_source("dk_log");
	
	if(!isset($argv)){
		$argv = func_get_args();
	}
	
    dk_log("NOTICE", $argv[0]);
}




###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
if(!function_exists('DKTEST')){ function DKTEST() {
	#dk_debugFunc 0

    dk_notice("test dk_notice message");
}}


include_once(str_replace("\\","/",$_SERVER['USERPROFILE'])."/DigitalKnob/Development/DKPhp/functions/DK.php");
?>